src/Routing/SignLoader.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DocusignBundle\Routing;
+
+use Symfony\Component\Config\Loader\Loader;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * @author Vincent Chalamon <vincentchalamon@gmail.com>
+ */
+final class SignLoader extends Loader
+{
+    private $config;
+
+    public function __construct(array $config)
+    {
+        $this->config = $config;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function load($resource, $type = null): RouteCollection
+    {
+        $routeCollection = new RouteCollection();
+
+        // Load static routes: callback & webhook
+        $routeCollection->add('docusign_callback', (new Route('docusign/callback', [
```

Comment:
callback is not static, it comes from `callback` configuration

Reply:
Not this one: this is the default callback.

In the configuration, the user will be able to set a `callback: https://www.google.fr` or a route name, like: `callback: foo_callback` or even `callback: docusign_callback`. In those cases, there is no need to create a route dynamically, just to create the `docusign_callback` default route once